multiqc/modules/base_module.py

```diff
@@ -162,7 +162,7 @@ def clean_s_name(self, s_name, root):
             root = ''
         # Split then take first section to remove everything after these matches
         for ext in config.fn_clean_exts:
-            s_name = s_name.split(ext ,1)[0]
+            s_name = os.path.basename(s_name.split(ext ,1)[0])
```

Comment:
As per your comment below - I can see how this can happen, great addition! Part of me wants you to move it out of the loop but it shouldn't make any difference :stuck_out_tongue: Nice!